socrates/views/layoutComponents.jade

```diff
@@ -50,15 +50,12 @@ mixin navbar(xs)
           +loginMenu
       ul.nav.navbar-nav
         li(data-nav-index=''): a(href='/', title='Home', style='padding-left: 5px') SoCraTes 2016
-        li(data-nav='schedule.html'): a(href='/schedule.html', title='Schedule')
-          span.glyphicon.glyphicon-calendar
-          span.hidden-sm  Schedule
+        li(data-nav='schedule.html'): a(href='/schedule.html', title='Format')
+          span.glyphicon.glyphicon-question-sign
```

Comment:
I'd rather keep the old icon (calendar) here. Better than questionmark

@NicoleRauch was meinst du?